Core/AppRuntime/Source/AppRuntime.cpp

```diff
@@ -44,6 +65,24 @@ namespace Babylon
                 {
                     func(env);
                 }
+                catch (const Napi::Error& error)
+                {
+                    std::ostringstream ss{};
+
+                    std::string msg = error.Message();
+                    std::string source = getStringPropertyFromError(error, "source");
+                    std::string url = getStringPropertyFromError(error, "url");
+                    int32_t line = getNumberPropertyFromError(error, "line");
+                    int32_t column = getNumberPropertyFromError(error, "column");
+                    int32_t length = getNumberPropertyFromError(error, "length");
+                    std::string stack = getStringPropertyFromError(error, "stack");
+
+                    ss << "Error on line " << line << " and column " << column
+                       << ": " << msg << ". Length: " << length << ". Source: " << source << ". URL: " << url << ". Stack:" << std::endl << stack << std::endl;
+                    
+                    Napi::Error newError = Napi::Error::New(env, ss.str());
```

Comment:
This doesn't belong here. They should go in the `DefaultUnhandledExceptionHandler` implementation instead.

Reply:
I'm not sure to follow. `DefaultUnhandledExceptionHandler` is platform specific. this code is not.

Reply:
Done with rethrow. JSC is ok btw.